projects/kernel-config/kernel_config.debug

```diff
@@ -24,3 +24,4 @@ CONFIG_KGDB_SERIAL_CONSOLE=y
 CONFIG_KGDBOC=y
 CONFIG_DEBUG_RODATA_TEST=y
 CONFIG_DEBUG_WX=y
+# CONFIG_PANIC_ON_OOPS is not set
```

Comment:
I assume that it's because you may want to inspect some state after the panic, but I don't really know. This change is just a minor simplification of what we were already doing: https://github.com/linuxkit/linuxkit/pull/1945/commits/fb0113481a5aa4d8fd7d1ef4f731096ad9fae63f